WalletWasabi.Gui/Controls/NoparaPasswordBox.cs

```diff
@@ -231,17 +232,21 @@ protected override async void OnKeyDown(KeyEventArgs e)
 							paste = true;
 						}
 					}
-				}
 
-				if (paste)
-				{
-					string text = await Application.Current.Clipboard.GetTextAsync();
-					if (!string.IsNullOrEmpty(text))
+					if (paste)
 					{
-						e.Handled = OnTextInput(text);
+						string text = await Application.Current.Clipboard.GetTextAsync();
+						if (!string.IsNullOrEmpty(text))
+						{
+							e.Handled = OnTextInput(text);
+						}
+
+						PaintText();
```

Comment:
'OnTextInput' contains 'PaintText();' - this can be removed.

Reply:
Done.